Add maintenance commands to README (#1540)
safe-global/safe-transaction-service: README.md

- [v1.3.0 L2](https://github.com/safe-global/safe-deployments/blob/main/src/assets/v1.3.0/gnosis_safe_l2.json)
- [Other related contracts and previous Safe versions](https://github.com/safe-global/safe-deployments/blob/main/src/assets)

## Service maintenance

Service can run into some issues when running in production:

### Indexing issues
You can tell there are indexing issues if:

> - Executed transactions are missing from the API (`all-transactions`, `multisig-transactions`, `module-transactions`... endpoints). If you
> use the **web UI** please make sure by checking **Transaction Service API**, as sometimes it can be a cache delay in
> the **Safe Client Gateway** which is between web UI and **Safe Transaction Service**.

Reviewer: ```suggestion
- Executed transactions are missing from the API (`all-transactions`, `multisig-transactions`, `module-transactions`... endpoints). If you use the [Safe{Wallet} Web client](https://github.com/safe-global/safe-wallet-web) you should check what is the current state of the Safe Client Gateway cache as it might have outdated data.
```

- Asset transfers (ERC20/721) are missing from `all-transactions` or `transfers` endpoints.

> - You see in the logs errors about an owner cannot be removed or similar inconsistent errors when `worker-indexer` is processing decoded data.

Reviewer: ```suggestion
- You see error logs such as "Cannot remove owner" or similar inconsistent errors when `worker-indexer` is processing decoded data.
```

There are multiple options for this. Connect to either `web` or `worker` instances. Running commands inside of `tmux` is recommended
(installed by default):
- `python manage.py check_index_problems`: it will try to automatically fix missing transactions.
Tokens related transactions (ERC20/721) will not be fixed with this method. This method will take a while, as it needs to compare
database data with blockchain data for every Safe.
- `python manage.py reindex_master_copies --from-block-number X --addresses 0x111 0x222`: if you know the first problematic block,
it's faster if you trigger a manual reindex. `--addresses` argument is optional, but if you know the problematic Safes providing

> them will make reindexing **way** faster, as if not all the Safes are reindexed.

Reviewer: ```suggestion
them will make reindexing **way** faster, as only those Safes will be reindexed (instead of the entire collection).
```

> If you see ERC20/ERC721 missing:

Reviewer: ```suggestion
If you see ERC20/ERC721 transfers missing:
```
